libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -904,6 +904,26 @@ void RC_Channel::do_aux_function(const aux_func_t ch_option, const aux_switch_po
         break;
 #endif // HAL_MINIMIZE_FEATURES
 
+    case AUX_FUNC::GOPRO_MODE_TOGGLE: {
```

Comment:
As mentioned elsewhere - this is a generic video/camera toggle function - we should label it that way so that @andyp1per can use it for RunCam too ;-)

Reply:
Well, if you call it video/camera, then it's also still not generic :).  It will then only apply for use cases where there only two modes are video and camera.  I can rename it camera mode toggle.  It will then be appropriate for any camera where the method is toggling through two or more modes.  If a camera has 5 modes, you're toggling from mode 1 to mode 2 to mode 3, to mode 4, to mode 5, to mode 1, and so on.  I think that is as generic as this can be.

If down the road, someone wants to make an aux switch function that is high for video, low for camera specifically, they're welcome to do that and it wouldn't contradict the toggle function of this.